scripts/deploy_example_gcp

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPTS_DIR/common"
+
+"$SCRIPTS_DIR/build_example" "hello_world"
+
+bazel_build_flags+=(
+    '--config=clang'
+)
+
+# Prerequisites:
+# - Google Cloud SDK
+# - Environment variable GCP_PROJECT_ID: id of the GCP project to use (eg oak-vm-testing).
+# - Environment variable GCP_ACCOUNT_FILE: path to the exported key for the service
```

Comment:
Could you add checks for non-emptiness of these variables?

Reply:
Added it to gcp_common script.